public/client.js

```
@@ -0,0 +1,534 @@
+            // Reset button
+            generateBtn.textContent = 'Generate & Calculate';
+            generateBtn.disabled = false;
+        }
+    });
+    
+    // Function to render spectrum chart
+    function renderSpectrumChart(frequencyBins, magnitudeValues, quantity) {
+        const ctx = document.getElementById('spectrumChart').getContext('2d');
+        
+        // Filter out frequencies outside the min/max range
+        const minFreq = parseFloat(document.getElementById('minFrequency').value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)